src/main/java/TaskManager.java

```diff
@@ -0,0 +1,41 @@
+public class TaskManager {
```

Comment:
I like that  the TaskManager is seperate from Duke. Makes the code more readable.